lib/impress.application.js

```diff
@@ -187,7 +189,7 @@ impress.application.mixin = (application) => {
   }
 
   application.createSandbox = (callback) => {
-    const sandbox = { module: {}, api: {} };
```

Comment:
Yes, it deprecates `module.exports` too because we have different modularity style, so I think no need to support node.js style or strange things like `__filename`, `__dirname` (may affect security too) and because applied scripts are supposed to know where they placed.